src/prog_models/prognostics_model.py

```diff
@@ -208,17 +208,15 @@ def __init__(self, **kwargs):
             if 'process_noise' not in self.parameters:
                 self.parameters['process_noise'] = 0.1
             else:
-                self.parameters['process_noise'] = self.parameters['process_noise'] # To force  __setitem__
+                self.parameters['process_noise'] = self.parameters['process_noise']         # To force  __setitem__
```

Comment:
What's the story with this change?

Reply:
This was to improve readability. It made it even with another related comment